### DISCUSSION TITLE
[✨] Add official Civet support to `create-qwik` package

### DISCUSSION THREAD

**Discussion by danielbayley** [QwikDev/qwik-evolution#126] [Proposals For Qwik]
### Is your feature request related to a problem?

Forcing the use of TypeScript and JSX to be able to take full advantage of Qwik, for those less than enthused with that syntax can be alleviated by it’s compatibility with [Civet](https://civet.dev), which I noticed you guys are [sponsoring](https://opencollective.com/civet) :)

### Describe the solution you'd like

Add official Civet support as prompt option to the [`create-qwik`](https://qwik.builder.io/docs/getting-started#create-an-app-using-the-cli) `init`ializer package… I suppose it should be an [ecosystem](https://qwik.builder.io/ecosystem)/[integration](https://qwik.builder.io/ecosystem#integrations).

### Describe alternatives you've considered

[CoffeeScript](https://github.com/jashkenas/coffeescript/issues/5307) -> TypeScript

### Additional context

QwikDev/qwik#2878, https://github.com/DanielXMoore/Civet/issues/858

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.

**Comment by brandonpittman** [comment]
Wondering if this would get Qwik support since Builder is sponsoring Civet.
